src/common/w25x.c

```diff
@@ -10,6 +10,7 @@
 #include <w25x.h>
 #include "main.h"
 
+#define PAGE_SIZE 256
```

Comment:
please avoid using #defines if not absolutely necessary

Reply:
In this case it's possible, fixed in the next push.